src/NServiceBus.Core/Hosting/Helpers/AssemblyScannerConfiguration.cs

```diff
@@ -24,7 +24,7 @@ public class AssemblyScannerConfiguration
         /// <summary>
         /// Defines whether nested directories should be included in the assembly scanning process. Default value is <code>false</code>.
         /// </summary>
-        public bool ScanAssembliesInNestedDirectories { get; set; } = false;
+        public bool ScanAssembliesInNestedDirectories { get; set; }
```

Comment:
>  Personally, I'd keep them as this is not just some local/private field but a public property which might be set from anywhere.

that makes no sense. the default just inits the field at construction time. "might be set from anywhere" has no bearing on the default